pkg/Microsoft.Private.PackageBaseline/packageIndex.json

```diff
@@ -185,16 +185,18 @@
         "2.2.0",
         "2.2.1",
         "2.2.2",
-        "3.0.0"
+        "3.0.0",
+        "3.1.0"
```

Comment:
This change is smaller than I expected.  I expected all packages to get a stable entry.  Perhaps you should run the target for AllConfigurations as well?

Reply:
Yeah. run the target with /p:BuildAllConfigurations=true.

Reply:
I went ahead and fixed this.